### PR TITLE
Create run.sh for Linux environment

### PR DESCRIPTION
**Pull request by AndresCdo**
Bash script to run Auto-GPT on Linux.

### Background
A certain percentage of developers run programs on Linux.

### Changes
Introduces Bash script to run python program.

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+# This script is used to run the main script in a virtual environment.
+if ! command -v python3 -m venv &> /dev/null    # Check if venv is installed and install it if not
+then
+echo "venv is not found. Installing venv..."
+sudo apt-get update
+sudo apt-get install python3-venv
+fi
+
+python3 -m venv env # Create a virtual environment
```
Comment (alexonab) [inline run.sh]: Maybe change env to .venv so as not to clutter the working directory.

File: run.sh [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    python3 -m venv venv && # Create a virtual environment
+    source venv/bin/activate # Activate the virtual environment
+}
+
+{ # try
+    set_venv # Set the virtual environment
+} || { # catch
+    echo "An error occurred while creating the virtual environment."
+    echo "venv is not found. Installing venv..."
+    sudo apt-get update
```
Comment (hdkiller) [inline run.sh:25]: When I run something I don’t expect that system packages will be installed. Also what about other distributions which are not using apt for package management?

**Comment by nponeccop** [comment]
@AndresCdo There are conflicts now

**Comment by richbeales** [comment]
we now have a run.sh in the application

**Comment by AndresCdo** [comment]
Closed.

**Pull request closed without merging**